CMakeLists.txt

```diff
@@ -1276,7 +1291,7 @@ else()
 
     if(UA_ENABLE_PUBSUB)
         list(APPEND UA_FILE_DATATYPES ${PROJECT_SOURCE_DIR}/tools/schema/datatypes_pubsub.txt)
-        if(UA_ENABLE_PUBSUB_INFORMATIONMODEL)
+        if(UA_ENABLE_PUBSUB_INFORMATIONMODEL OR UA_ENABLE_PUBSUB_MQTT_METADATA)
```

Comment:
Why does metadata publishing require the nodeset xml?
Add a short comment to explain.

Reply:
In fact it is not necessary. I removed it.